packages/sdk/app-framework/src/plugins/common/layout.ts

```diff
@@ -95,6 +97,7 @@ export enum LayoutAction {
   SET_LAYOUT = `${LAYOUT_ACTION}/set-layout`,
   SET_LAYOUT_MODE = `${LAYOUT_ACTION}/set-layout-mode`,
   SCROLL_INTO_VIEW = `${LAYOUT_ACTION}/scroll-into-view`,
+  UPDATE_PLANK_SIZE = `${LAYOUT_ACTION}/update-plank-size`,
```

Comment:
could this be framed as a layout update somehow? the idea of these common definitions is that they aren't coupled to our implementation